### ISSUE TITLE
Unable to sync the blockchain

### ISSUE THREAD

**Issue by b00f** [pactus-project/pactus#900]
## Description

Unable to sync the blockchain in Testnet at block 35861. Restarting the node won't fix the issue.

```
WRN committing block failed, removing block from the cache error="invalid public key: public key is not set" _sync="{☍ 30 ⛃ 240}" height=35861
```

### What Happened

Fixing #869 has backfired. Block 35861 contains a Bond transaction for a bootstrap validator. The bootstrap validator has a public key, but at the same time, the `LastBondingHeight` for the bootstrap validator has been set to zero.

We can fix this by adding a new method to the Sandbox to determine if the Validator is sandboxed or not. However, this will add more complexity to the code. The issue #869 is rare and occurs only once in Testnet. This issue is not permanent; for example, after one day (once the first transaction expires), it will be fixed, or users can broadcast the transaction to other nodes. Fixing both issues will increase the code complexity and make the code harder to understand.

It's better to revert the fix for #869.

**Comment by b00f** [comment]
We can monitor the Testnet and if the issue #869 occures more, we can decide for a better fix.
